docs/swagger.json

```diff
@@ -0,0 +1,882 @@
+{

```

Comment:
I doubt this is a javalin fresh generated OAS. Please add the newly generated one